addons/osio-addon/src/test/java/io/fabric8/launcher/osio/tenants/TenantIT.java

```diff
@@ -39,7 +39,7 @@
 
     @ClassRule
     public static HoverflyRule witVirtualization = createHoverflyProxy("wit-simulation.json",
-                                                                       "api.openshift.io|api.prod-preview.openshift.io");
+                                                                       "api.openshift.io|api.prod-preview.openshift.io", 8558);
```

Comment:
Isn't this supposed to be using a random port?

Reply:
Not until I remove arquillian on this one in the #218